content/help/faq/security/k8s-health-checks.md

```diff
@@ -13,10 +13,12 @@ communicate with the service with http traffic.
 
 You can use a separate port for health check and enable mutual TLS only
 on the regular service port. Refer to [Health checking of Istio
-services](/docs/tasks/traffic-management/app-health-check/)
-for more information.
+services by App Prober Rewrite](/help/ops/setup/app-health-check/) for more information.
```

Comment:
The title of the page is just "Health Checking of Istio Services"

Reply:
Done.